doc/source/conf.py

```diff
@@ -244,6 +244,11 @@ def __repr__(self):
     '    from pyvista import set_plot_theme\n'
     '\n'
     '    set_plot_theme("document")\n',
+    '\n'
+    'import vtkmodules\n'
+    'if not hasattr(vtkmodules "vtkEGLRenderWindow")\n'
+    '    # vtk is not egl.\n'
+    '    pv.start_xvfb()\n'
```

Comment:
I think we may just want to do this for tomorrow

```suggestion
'pv.start_xvfb()\n'
```